cirq-core/cirq/sim/clifford/stabilizer_state_ch_form.py

```diff
@@ -273,3 +273,32 @@ def project_Z(self, q, z):
             self.omega /= np.sqrt(2)
 
         self.update_sum(t, u, delta=delta)
+
+    def kron(self, other: 'cirq.StabilizerStateChForm') -> 'cirq.StabilizerStateChForm':
```

Comment:
This is covered indirectly by the simulator tests, but it would be better to have direct coverage.

Reply:
There are no tests on CH itself https://github.com/quantumlib/Cirq/issues/3021. And this may make sense: the CH form is cryptic and any tests on it would have to be equally cryptic. I believe testing through the simulator API surface makes sense in this case.